create_package.sh

```diff
@@ -22,8 +22,9 @@ mkdir -vp /ovms_release/lib
 mkdir -vp /ovms_release/lib/hddl/config
 mkdir -vp /ovms_release/lib/custom_nodes
 
-if [ -f /openvino_tokenizers/build/src/libopenvino_tokenizers.so ]; then cp -v /openvino_tokenizers/build/src/libopenvino_tokenizers.so /ovms_release/lib/ ; fi
-if [ -f /openvino_tokenizers/build/src/libcore_tokenizers.so ]; then cp -v /openvino_tokenizers/build/src/libcore_tokenizers.so /ovms_release/lib/ ; fi
+tokenizers_path=/ovms/bazel-out/k8-opt/bin/external/llm_engine/llm_engine_cmake_ubuntu/lib/
```

Comment:
This way we cannot use tokenizers for images without mediapipe @dtrawins @rasapala @mzegla 
It means users cannot load single models (tokenizers) with OVMS without mediapipe